src/dotnet/Chat/Module/ChatSettings.cs

```diff
@@ -4,4 +4,6 @@ public class ChatSettings
 {
     public string Db { get; set; } = "";
     public string Redis { get; set; } = "";
+
+    public TimeSpan TurnOffRecordingAfterIdleTimeout { get; set; } = TimeSpan.FromMinutes(2);
```

Comment:
Should go to audio module & be `IdleRecordingTimeout`.

Reply:
done